src/docs/admin/references/NotificationServiceReference.adoc

```diff
@@ -0,0 +1,195 @@
+==== Housekeeping
+
+Notifications in the notification service have a global life span defined in the
+application.properties located in * `PROACTIVE_HOME/dist/war/notification-service/WEB-INF/classes/application.properties`
+Passed that time, they will be removed from the database.
+
+Can be defined:
+
+- The call frequency of the housekeeping defined by a CRON expression:
+----
+# This cron expression determines the housekeeping call frequency.
+# Default value is 10 minutes: this will invoke the housekeeping mechanism
+# to remove every notifications whose scheduled removal time is up.
+notifications.housekeeping.cronexpression=*/10 * * * *
+----
+
+- The life span of the notifications in seconds:
+----
+# Automatic remove notification delay (in seconds).
+# The time at which notifications will be stored in the database
+# Set this time to 0 if you don't want the notifications to be removed.
+# Default value is 604800 seconds = 1 week
+notifications.housekeeping.removedelay=604800
+----
+
+[[_notification_methods]]
+==== Notification methods
+
+[[_email_notification_method]]
+===== Email
+
+Email notification method uses SMTP protocol to send emails, thus the notification
```

Comment:
The *Email notification method* uses the  Simple Mail Transfer Protocol to send emails. The notification service requires a valid SMTP configuration in order to send emails.

These settings are specified in the properties file located at: